ui/components/app/wallet-overview/eth-buttons.tsx

```diff
@@ -0,0 +1,379 @@
+import React, { useContext } from 'react';
```

Comment:
This file just extracted the buttons from `eth-overview.js`.  No changes other than converted to typescript.